docs/pom.md

```diff
@@ -0,0 +1,59 @@
+# Page Object Models
+Large test suites can be structured to optimize ease of authoring and maintenance.
+Page object models are one such approach to structure your test suite.
+
+<!-- GEN:toc-top-level -->
+- [Introduction](#introduction)
+- [Implementation](#implementation)
+<!-- GEN:stop -->
+
+## Introduction
+A page object represents a part of your web application. An e-commerce web
+application might have a home page, a listings page and a checkout page. Each of
+them can be represented by page object models.
+
+Page objects **simplify authoring**. They create a higher-level API which suits
+your application.
+
+Page objects **simplify maintenance**. They capture element selectors in one place
+and create reusable code to avoid repetition.
+
+## Implementation
+Page object models wrap over a Playwright [`page`](./api.md#class-page).
+
+```js
+// in models/Search.js
+class SearchPage {
+  constructor(page) {
+    this.page = page;
+  }
+  async navigate() {
+    await this.page.goto('https://bing.com');
+  }
+  async fill(text) {
+    await this.page.fill('[aria-label="Enter your search term"]', text);
+  }
+}
+module.exports = { SearchPage };
+```
+
+Page objects can then be used inside a test.
+
+```js
+// in search.spec.js
```

Comment:
This comment hints that we see a test file, but content does not. Maybe just a short snippet for now, without requires/launch?

Reply:
Sounds good. Thanks for the review!